ui/src/experiments/containers/DetailPageContainer.jsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Map, List } from 'immutable';
+import { Row, Col } from 'antd';
+
+import { LinkOutlined } from '@ant-design/icons';
+import fetchExperiment from '../../actions/experiments';
+import withRouteActionsDispatcher from '../../common/withRouteActionsDispatcher';
+import ContentBox from '../../common/components/ContentBox';
+import DocumentHead from '../../common/components/DocumentHead';
+import AffiliationList from '../../common/components/AffiliationList';
+import { SEPARATOR_MIDDLEDOT } from '../../common/components/InlineList';
+import RelatedRecordsList from '../../common/components/RelatedRecordsList';
+import RequireOneOf from '../../common/components/RequireOneOf';
+import RichDescription from '../../common/components/RichDescription';
+import DeletedAlert from '../../common/components/DeletedAlert';
+import UrlsAction from '../../literature/components/UrlsAction';
+import ExperimentDates from '../components/ExperimentDates';
+import ExperimentCollaboration from '../components/ExperimentCollaboration';
+import ExperimentAssociatedArticles from '../components/ExperimentAssociatedArticles';
+import ExperimentCollaborationMembers from '../components/ExperimentCollaborationMembers';
+import ExperimentCollaborationArticles from '../components/ExperimentCollaborationArticles';
+import PublicNotesList from '../../common/components/PublicNotesList';
+import { EXPERIMENT_PAPERS_NS } from '../../reducers/search';
+import { newSearch } from '../../actions/search';
+import ExperimentPapers from './ExperimentPapers';
+import { makeCompliantMetaDescription } from '../../common/utils';
+
+function DetailPage({ record }) {
+  const metadata = record.get('metadata');
+
+  const legacyName = metadata.get('legacy_name');
+  const institutions = metadata.get('institutions', List());
```

Comment:
why default was necessary? you could do `institutions &&` rather than `institutions.size > 0`